backend/tournesol/suggestions/suggester.py

```diff
@@ -0,0 +1,236 @@
+from django.db.models import Q, QuerySet
+
+from core.models import User
+from tournesol.models import ComparisonCriteriaScore, Entity, Poll
+from tournesol.suggestions.graph import Graph
+from tournesol.suggestions.suggested_user import SuggestedUser as RecommendationUser
+from tournesol.suggestions.suggested_video import SuggestedVideo
+
+
+class Suggester:
+    """
+    An interface class to ask for suggestions of videos to compare
+    """
+    # Entity to specific video structure dictionary
+    _entity_to_video: dict[Entity, SuggestedVideo]  # todo Use only uid as key
+    _comparison_reference_video: SuggestedVideo
+    # Graph containing all videos and existing comparisons, used to get the video preferences
+    _complete_graph: Graph
+    # Dictionary linking a user to its comparison graph, used to get its information gain
+    _user_specific_graphs: dict[User, Graph]
```

Comment:
`user.id` can be used as keys instead of storing the whole User instance.

Reply:
Done :)